### PR TITLE
restructure project and remove unecessary files to allow dev-server reloading

### PR DESCRIPTION
**Pull request by kofiarkoh**
So the project had issues with the reloading the file server when code changes. so i restructed it and added flask blueprints to make it work seamlessly!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,7 +206,7 @@ Install the required packages by running the following command in the terminal
 Step 4:
 Run the following command in the terminal
 
-    python application.py
+    export FLASK_APP=FLASK_APP=fitnessapp && export FLASK_ENV=development && FLASK_DEBUG=1 && flask run
```
Comment (lingjunliu) [inline README.md:209]: "FLASK_APP=" repeated

Comment (kofiarkoh) [reply README.md:209]: will correct that.

Comment (lingjunliu) [reply README.md:209]: I have corrected it in my branch

Comment (lingjunliu) [reply README.md:209]: preparing the pull request for event creation

**Pull request merged**